src/crimson/os/seastore/seastore.h

```diff
@@ -208,7 +223,9 @@ class SeaStore final : public FuturizedStore {
 	  }).safe_then([&ret](auto _ret) {
 	    ret = _ret;
 	  });
-	}).safe_then([&ret] {
+	}).safe_then([&ret, optype, &t, this]{
```

Comment:
please add a space before `{`.

Reply:
instead of using `const_cast`, how about just adding the `mutable` specifier?

Reply:
mutable doesn't work, still has error: passing ‘const crimson::os::seastore::SeaStore’ as ‘this’ argument discards qualifiers [-fpermissive]